snapshot-labs/sx-monorepo
# feat: add multiple treasuries support in editor

### Summary

This PR adds support for configuring execution with multiple treasuries 

Closes: https://github.com/snapshot-labs/sx-monorepo/issues/138

### How to test

1. Get two spaces to test with (one on offchain with oSnap enabled and multiple treasuries configured and another onchain with execution and treasury configured).
2. Try creating proposals on both.
3. Everything works as expected: on offchain you can select multiple treasuries at once, onchain should be limited to one at the time.
4. You can open drafts with execution.
5. You can open editor with execution from treasury page by clicking Send token or Send NFT button.

### Screenshots

<img width="642" alt="image" src="https://github.com/user-attachments/assets/87f4ffde-d49a-428e-ab68-d6e8f94c1802">
<img width="628" alt="image" src="https://github.com/user-attachments/assets/981c075c-e6c1-4461-8975-561a5a18ed91">
